sources/packages/backend/libs/src-sql/sfas-restrictions/Bulk-insert-restrictions.sql

```diff
@@ -0,0 +1,120 @@
+/* 
+ * After all the SFAS restrictions are imported, 
+ * the below query inserts one Legacy restriction entry 
+ * per student if they have one or more restrictions 
+ * imported from the sfas system that are not present 
+ * in the sims restrictions.
+ */
+INSERT INTO
+  sims.student_restrictions (student_id, restriction_id)
```

Comment:
As discussed, the audit user can be received as parameter and inserted to `sims.student_restrictions`